### ISSUE TITLE
Could not find module `ember-compatibility-helpers` imported from `ember-decorators/utils/decorator-macros`

### ISSUE THREAD

**Issue by juwara0** [pzuraq/ember-compatibility-helpers#7]
### Problem Description

I have an addon `ember-frost-core` that I am consuming `ember-decorators` as a dependency. https://github.com/ciena-frost/ember-frost-core/blob/38d74e039bc91686c322959dea00f9bccb41b2aa/package.json#L101

When I consume `ember-frost-core` in another addon `ember-frost-popover` (https://github.com/juwara0/ember-frost-popover/blob/upgradeEmberCli_2-16-2/package.json#L68) that is not using `ember-decorators` but I see it in the node_modules tree:

```->  npm ls ember-compatibility-helpers
ember-frost-popover@4.4.1 /Users/Micarrol/Devel/ember-frost-popover
└─┬ ember-frost-core@2.0.0
  └─┬ ember-decorators@1.3.0
    └── ember-compatibility-helpers@0.1.1 
```
I am getting an error in my console:
```
Uncaught Error: Could not find module `ember-compatibility-helpers` imported from `ember-decorators/utils/decorator-macros
```

**Comment by pzuraq** [comment]
Fixed by #8

**Comment by juwara0** [comment]
Thank you for taking a look at this issue so quickly. I verified that this issue is now fixed in the codebase that I saw the problem in.

**Comment by Patsy-issa** [comment]
@pzuraq 👋  any chance to get a release with the fix from #8, currently breaking our addon builds that have a dependency on `ember-decorators` > 1.2.1.

**Comment by pzuraq** [comment]
This fix was released, could be that the fix didn’t take though. Can you confirm that you are using v0.1.2?

**Comment by Patsy-issa** [comment]
Ah my bad, this fix was indeed released, the problem we are facing was then fixed in #9 with the init hook refactor, that is the commit that we are missing.

**Comment by pzuraq** [comment]
Updated the hooks with a better solution and released v0.1.3. Let me know if you're still having that issue!
